vars/FreeBSD.yml

```diff
@@ -1,3 +1,11 @@
+__argus_radium_user: argus
+__argus_radium_user_home: /var/log/argus
+__argus_radium_user_shell: /usr/sbin/nologin
+__argus_radium_user_uid: 561
```

Comment:
* FreeBSD's package should reserve UID/GID. is there any reason that argus-clients does not?
* creating users/groups prior to the package installation does not work for Red Hat variants?
* i think creating users/groups with specific IDs is better but without ID is acceptable when  it is difficult.
* BTW, the role specifies UID/GID in tasks/main.yml and i may not understand what 'without specific ID' means.
* s/561/1002/ just means that the role uses 1002 for FreeBSD in tests/serverspec/ and i thought the default value is supposed to be 1002.
* In addition, 561 is reserved in UIDs/GIDs and other numbers should be the default.